checks.d/docker_daemon.py

```diff
@@ -96,12 +96,21 @@ def image_tag_extractor(c, key):
     return None
 
 
+def container_name_extractor(c):
```

Comment:
I'd expect it to return directly a single name, instead of a list. So that we can `container_name_extractor(container)` instead of `container_name_extractor(container)[0]`.